.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kevinushey what's your take on this? I definitely had to use Eclipse to develop this set of changes so I'm planning to keep the Eclipse project in my working directory; however, in the short term, you are probably the only other person making changes to the Scala code. So, have you been using Eclipse already for your Scala changes? If not, would having an Eclipse project be helpful to you? If the answer is no, then I don't think it's worth checking this in, otherwise, I'll parameterize the paths to make you can use this as well.